.gitignore

```diff
@@ -20,5 +20,3 @@ ledger/3.0.0mainnet
 ledger/berkeley-devnet
 mina-workdir
 .idea/
-
```

Comment:
Could this comment be squashed with the commit introducing the change? The initial change should simply not appear in the git history.